sys/vfs_syscalls.c

```diff
@@ -60,7 +60,7 @@ int do_lseek(thread_t *td, int fd, off_t offset, int whence) {
   /* TODO: Whence! Now we assume whence == SEEK_SET */
```

Comment:
I see... so please add corresponding issue to tracking system and merge this PR as is.